static/css/restyle/restyle.less

```diff
@@ -226,6 +226,10 @@ button.link:active span {
   width: 78%;
 }
 
+.secondary {
+  width: 21.49%
+}
+
```

Comment:
There already is `.secondary` in https://github.com/mozilla/addons-server/blob/fix-float-issues-2163/static/css/restyle/restyle.less#L931 - maybe let's use that?